bql/planner/data_access.go

```diff
@@ -508,6 +512,15 @@ func objectToCell(o *triple.Object) (*table.Cell, error) {
 	return nil, fmt.Errorf("unknown object type in object %q", o)
 }
 
+type skippableError struct {
+	situation string
+	err       error
+}
```

Comment:
It is often preferable to have the types at the top of the file, could you move this there?
And this seems like a nice solution, thanks=]

Reply:
Also, sometimes we declare:
var _ error = (*skippableError)(nil)
at the top level to ensure and "document" this is a valid implementation of the interface

Reply:
Thank you Kloss. I have just done so. 😀